worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageMappingCallable.java

```diff
@@ -36,7 +36,6 @@
 
     private final int year;
 
-    private final boolean skipBillablePeriodCheck;
```

Comment:
Removing this line will trigger changes in the ab2d-ops repo. We need to capture that in a ticket.

Reply:
It's definitely used other places if that makes a difference. It's just not used in the Callable to determine the year to pass to the contract coverage search.